### DISCUSSION TITLE
Installing Vim Plugins

### DISCUSSION THREAD

**Discussion by wilful** [LazyVim/LazyVim#4401] [Q&A]
How can I install a plugin like https://github.com/rodjek/vim-puppet? This plugin is not available in Extras, and for some reason, my init.vim is not being read at startup. Thank you.

**Comment by cofeek-codes** [comment]
@wilful You can just create a file in `lua/plugins/` i.e. `lua/plugins/puppet.lua` with following content 
```lua
return {
  {
    "rodjek/vim-puppet",
  },
}
```
This will install this plugin and activate it on startup
![image](https://github.com/user-attachments/assets/f9a8ef12-7777-4d4c-8fb5-7a2cebb5a9bf)
